roofit/roofitcore/inc/RooProfileLL.h

```diff
@@ -62,7 +59,7 @@ class RooProfileLL : public RooAbsReal {
   TIterator* _piter ; //! Iterator over profile likelihood parameters to be minimized 
   TIterator* _oiter ; //! Iterator of profile likelihood output parameter(s)
```

Comment:
When you see (mutable) iterator members that are not serialised (`//!`), that is almost always a sign that a range-based for loop or a counting loop should be used. Less members, faster, easier to understand.

Just saying, maybe for another day ...